src/App.jsx

```diff
@@ -0,0 +1,89 @@
+import './App.css';
+import Card from './components/card';
+import Categories from './components/categories';
+import logo from './Images/logo.png';
+import cart from './Images/cart.png';
+import Sosedi from './Images/sosedi.png';
+import evroopt from './Images/evroopt.png';
+import green from './Images/green.png';
+import emall from './Images/emall.png';
+import vitalur from './Images/vitalur.png';
+
+function App() {
+  return (
+    <div>
```

Comment:
Instead of `div` we can use here [React.fragment](https://reactjs.org/docs/fragments.html)